packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -344,495 +344,559 @@ class InteractiveGraphEditor extends React.Component<Props> {
         }
 
         return (
-            <View>
-                <LabeledRow label="Answer type:">
-                    <GraphTypeSelector
-                        graphType={
-                            this.props.graph?.type ??
-                            InteractiveGraph.defaultProps.graph.type
-                        }
-                        // TODO(LEMS-2656): remove TS suppression
-                        onChange={
-                            ((
-                                type: Required<InteractiveGraphProps>["graph"]["type"],
-                            ) => {
-                                this.props.onChange({
-                                    graph: {type},
-                                    correct: {type},
-                                });
-                            }) as any
-                        }
-                    />
-                </LabeledRow>
-                <InteractiveGraphDescription
-                    ariaLabelValue={this.props.fullGraphAriaLabel ?? ""}
-                    ariaDescriptionValue={
-                        this.props.fullGraphAriaDescription ?? ""
-                    }
-                    onChange={this.props.onChange}
-                />
-                <InteractiveGraphCorrectAnswer equationString={equationString}>
-                    {graph}
-                </InteractiveGraphCorrectAnswer>
-                {this.props.correct?.type === "point" && (
-                    <LabeledRow label="Number of Points:">
-                        <GraphPointsCountSelector
-                            numPoints={this.props.correct?.numPoints}
-                            onChange={(points) => {
-                                this.props.onChange({
-                                    correct: {
-                                        type: "point",
-                                        numPoints: points,
-                                    },
-                                    graph: {
-                                        type: "point",
-                                        numPoints: points,
-                                    },
-                                });
-                            }}
-                        />
-                    </LabeledRow>
-                )}
-                {this.props.correct?.type === "angle" && (
-                    <>
-                        <View style={styles.row}>
-                            <Checkbox
-                                label={
-                                    <LabelSmall>Show angle measures</LabelSmall>
-                                }
-                                checked={
-                                    // Don't show indeterminate checkbox state
-                                    !!this.props.correct?.showAngles
+            <Id>
```

Comment:
Using wonder-blocks-core `Id` here since it's a class component. If it were a functional component, I would have used the `React.useId()` hook and avoided all this indentation 🥲